src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+// there are some figures
+public abstract class Figure {

```

Comment:
```suggestion
public abstract class Figure implements implements AreaCalculator, FigurePainter {
```
Можна ще на цьому етапі задекларувати інтерфейси і вони за замовчуванням поширяться на всіх фігур наслідників